### DISCUSSION TITLE
How can I control the canvas sizing?

### DISCUSSION THREAD

**Discussion by alonshoshani** [k-yle/rtsp-relay#137] [Q&A]
How can I control the canvas sizing?

**Comment by k-yle** [comment]
You can just use css right? `<canvas id="canvas" style="width: 1920px; height: 1080px"></canvas>`

**Reply by alonshoshani** [reply]
Yes but the stream resolution larger from the Canvas.

**Reply by k-yle** [reply]
you can always use css to resize the `<canvas />` element. If you want to save bandwidth, you can scale down the video before relaying it. 

To scale down the video, use [this ffmpeg flag](https://trac.ffmpeg.org/wiki/Scaling). Example:
```ts
app.ws('/api/stream', proxy({ additionalFlags: ['-vf', 'scale=720:480'] }));
```
